lib/sql-parser/src/base/sql/BaseSQL.js

```diff
@@ -327,6 +353,20 @@ class BaseSQL {
         if (this.hasWhereCondition)
 
             for (let key in this.where) {
+                /**
+                 * In Valid json object we can use dot between TABLE NAME and TABLE COLUMN NAME
+                 * Example:
+                 *    SELECT * FROM users,books WHERE users.id = books.userId
+                 *    The sql code above is the result of this json object
+                 *    @example
+                 *    {
+                 *        from: ['users', 'books'],
+                 *        where: {
+                 *            users$id: books$userId
+                 *        }
+                 *    }
+                 */
+                key = key.replace('$', '.');
```

Comment:
Issue in [this](https://github.com/treegex/opensql/pull/85/files#r1041334955) code review